src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicClientTransformer.java

```diff
@@ -359,17 +359,7 @@ private StaticLangSettingsView generateSettingsClass(GapicInterfaceContext conte
     settingsClass.batchingDescriptors(batchingTransformer.generateDescriptorClasses(context));
     // Remove additional retry codes, added due to the change in retry when using proto annotations.
     List<RetryCodesDefinitionView> retryCodes =
-        retryDefinitionsTransformer
```

Comment:
this was supposed to be in https://github.com/googleapis/gapic-generator/pull/2724 in order for us to not have to specify default retry code names